build-system/global-configs/canary-config.json

```diff
@@ -19,5 +19,6 @@
   "3p-frame-context-in-name": 1,
   "amp-accordion-session-state-optout": 1,
   "amp-playbuzz": 1,
-  "amp-selector": 1
+  "amp-selector": 1,
+  "sentinel-name-change": 1,
```

Comment:
please remove trailing `,` since this is a json file this will probably blow up the build